### PR TITLE
feat: CI setup

### LINKED ISSUE #5: Setting up CI
### Description

The project needs a Continuous Integration setup to ensure that this repo builds successfully for every iteration.

### PR DESCRIPTION
**Pull request by cs1137195420**
CI setup to build CMake fies automatically.

Fix: https://github.com/casbin-cpp/casbin-CMake-setup/issues/5

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    steps:
+      - name: Checkout
+        id: checkout
+        uses: actions/checkout@v2
+      - name: Configuring CMake files
+        id: building-files
+        run: |
+          mkdir build
+          cd build
+          cmake .. -DCMAKE_BUILD_TYPE=Release
```
Comment (EmperorYP7) [inline .github/workflows/ci.yml:32]: How about running the executable as well?

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (cs1137195420) [reply .github/workflows/ci.yml:32]: That's great. I would have a try.

**Comment by cs1137195420** [comment]
@EmperorYP7 I've updated the file ci.yml. Does it meet the requirement?

**Comment by hsluoyz** [comment]
@cs1137195420 plz always keep one commit in your PR:

image

**Comment by cs1137195420** [comment]
Sorry, I will create a new PR for this question.

**Comment by hsluoyz** [comment]
@cs1137195420 don't re-create PR, use this PR instead. Plz open it.

**Comment by cs1137195420** [comment]
> @cs1137195420 don't re-create PR, use this PR instead. Plz open it.

Wait a minute, I am trying to reopen it.

**Comment by cs1137195420** [comment]
@hsluoyz I tried but failed. My latest commit was 10 minutes ago, but it didn't appear in this PR. It seems that the File changed module doesn't work properly, which says the diff is unavailable:
image
Besides, the button of Reopen and comment is disabled. I cannot reopen this PR through clicking the button:
image
What should I do next? Is there any other way to reopen PR?

**Comment by hsluoyz** [comment]
@cs1137195420 OK, you can create a new PR this time, but remember to update the existing PR when something needs to be changed.

**Comment by cs1137195420** [comment]
Alright, this time I'll create a new PR to the issue after checking it out.

**Pull request closed without merging**